lib/stm32/f4/rcc.c

```diff
@@ -49,6 +49,8 @@ uint32_t rcc_ahb_frequency = 16000000;
 uint32_t rcc_apb1_frequency = 16000000;
 uint32_t rcc_apb2_frequency = 16000000;
 
+#define rcc_hsi_3v3 rcc_hse_16mhz_3v3
```

Comment:
Just commented on this in the IRC channel, but here it is again for the
record:

The f4 HSI always runs at 16MHz, so instead of replicating the 16MHz
presets for rcc_hsi_3v3, I aliased rcc_hsi_3v3 to rcc_hse_16mhz_3v3

On 09/05/2016 11:33 AM, Karl Palsson wrote:
>> @@ -49,6 +49,8 @@ uint32_t rcc_ahb_frequency = 16000000;
>>  uint32_t rcc_apb1_frequency = 16000000;
>>  uint32_t rcc_apb2_frequency = 16000000;
>>  
>> +#define rcc_hsi_3v3 rcc_hse_16mhz_3v3
> 
> what's this for?
>